frontend/integration-tests/tests/project-status.scenario.ts

```diff
@@ -0,0 +1,29 @@
+import { browser, $, $$, by, ExpectedConditions as until, Key, element } from 'protractor';
+import { appHost, testName, checkLogs, checkErrors } from '../protractor.conf';
+import { execSync } from 'child_process';
+
+import * as yamlView from '../views/project-status.view';
+import * as namespaceView from '../views/namespace.view';
+import * as projectStatusView from '../views/project-status.view';
+
+
+describe('Status page for all projects', () => {
+
+    it('check project selection list for all projects', async() => {
+      await browser.get(`${appHost}/overview/all-namespaces`);
+      await browser.wait(until.presenceOf(namespaceView.namespaceSelector));
+      await namespaceView.namespaceSelector.click();
+      await browser.wait(until.presenceOf(namespaceView.selectedNamespace));
+      await namespaceView.selectedNamespace.click();
+      expect(namespaceView.selectedNamespace.getText()).toEqual('all projects');
+    });
+     
+    it('Check Software Info on the right side', async() => {
```

Comment:
There's a good chance we will be removing this. It would be better to test the version in the about dialog in my opinion.

Reply:
@spadgett Thanks for hint, do you suggest to keep the checkpoint or remove the check here? Will add a new scenario for checking masthead pages.